proto/vtadmin.proto

```diff
@@ -72,6 +72,8 @@ service VTAdmin {
     // GetTablet looks up a tablet by hostname across all clusters and returns
     // the result.
     rpc GetTablet(GetTabletRequest) returns (Tablet) {};
+    // PingTablet checks that the specified tablet is awake and responding to RPCs. This command can be blocked by other in-flight operations.
+    rpc PingTablet(PingTabletRequest) returns (PingTabletResponse) {};
```

Comment:
nitpicking 🙈 can you keep this alphabetized?

Reply:
Absolutely!!